src/Fhir.Anonymizer.Shared.FunctionalTests/TestResources/patient-substitute-complex-target.json

```diff
@@ -0,0 +1,185 @@
+﻿{
+  "resourceType": "Patient",
```

Comment:
Maybe we should add test inputs for bundle and contained?

Reply:
Thanks. Added.